src/plutus_gen/code_emitters_plutus.rs

```diff
@@ -1,7 +1,8 @@
+use crate::plutus_gen::extraction::compile_plutus_expressions;
```

Comment:
I guess `code_emitters_plutus.rs` pertains to Plinth? rename the file accordingly